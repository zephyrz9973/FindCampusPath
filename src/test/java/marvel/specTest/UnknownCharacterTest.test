#Test Case with unknown character

#Create the graph
LoadGraph graph1 marvelTest.tsv

#find path for unknown node
FindPath graph1 Rabbit Cat