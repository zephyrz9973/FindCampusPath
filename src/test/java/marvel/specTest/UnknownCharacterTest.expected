#Test Case with unknown character

#Create the graph
loaded graph graph1

#find path for unknown node
unknown character Rabbit